packages/ocaml-options-only-fp/ocaml-options-only-fp.1/opam

```diff
@@ -15,6 +15,7 @@ conflicts: [
   "ocaml-option-nnpchecker"
   "ocaml-option-address-sanitizer"
   "ocaml-option-leak-sanitizer"
+  "ocaml-option-tsan"
```

Comment:
Actually, there should be no incompatibility between fp and tsan.

Reply:
this package is an `ocaml-options-only-*` package which explicitly requires to only have `fp` and no other options.